org.idempiere.test/src/org/idempiere/test/form/CreateFromStatementFormTest.java

```diff
@@ -166,7 +166,7 @@ public void loadPayments(Integer BankAccount, Integer BPartner, String DocumentN
 				Timestamp DateFrom, Timestamp DateTo, BigDecimal AmtFrom, BigDecimal AmtTo, Integer DocType,
 				String TenderType, String AuthCode) {
 			Vector<Vector<Object>> datas = super.getBankAccountData(BankAccount, BPartner, DocumentNo, DateFrom, DateTo, AmtFrom, AmtTo, DocType,
-					TenderType, AuthCode);
+					TenderType, AuthCode, 0);
```

Comment:
Same here, this unit test fails because `getBankAccountData` does not return any data

Reply:
Fixed Unit tests

Reply:
@PeterTakacs300 pls re-test it.